hack/verify/validate-code-format.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -e
+
+FILES=$(gofmt -s -l pkg cmd)
+
+if [ -n "$FILES" ]; then
+    echo You have go format errors in the below files, please run "gofmt -s -w pkg cmd"
```

Comment:
Might wanna add single quotes around this whole line, as written those double quotes won't print.